jvcl/run/JvDriveCtrls.pas

```diff
@@ -1686,7 +1686,11 @@ procedure TJvFileListBox.SetDirectory(const Value: string);
   begin
     inherited ApplyFilePath(Value);
     if Assigned(FChangeNotify) and (FChangeNotify.Notifications.Count > 0) then
+    begin
+      FChangeNotify.Active := false;
```

Comment:
As always when Disabling/Enabling, a `try..except`block is in order
Also, `False` takes a initial capital letter

Reply:
I can understand the desire for a try/except block, but:
a) what shall it cover? A single block covering all three lines from Active := False to Active := True?
b) what would I need to put into the except in this case? leaving it empty is a no go as well.
    Should I raise some exception? But if yes which one? And what would be the benefit? A more 
    descriptive message compared to the default exception I'd replace this way?

Reply:
My bad, I was thinking `try..finally` but was carried away by another code I was reviewing at the same time.
So basically, a `try` after that line, the next one before the `finally` and the setting back to `Active := True` in the `finally` part.

Reply:
I'm not 100% sure I understood you completely. But I put in a try/finally block now. Is this ok now or what is missing?

Reply:
Yes, that's perfectly fine.